package.json

```diff
@@ -11,6 +11,9 @@
     "tree-sitter",
     "javascript"
   ],
+  "dependencies": {
```

Comment:
in default package.json when I deleted the package.json and did `tree-sitter generate`